src/Documents.ts

```diff
@@ -436,6 +436,28 @@ class Documents implements Pointer.Library, Pointer.Validator, ObjectSchema.Reso
 		return HTTP.Request.Service.delete( documentURI, body, requestOptions );
 	}
 
+	removeAllMembers( documentURI:string, requestOptions:HTTP.Request.Options = {} ): Promise<HTTP.Response.Class> {
+		if( !! this.context ) documentURI = this.context.resolve( documentURI );
+		let containerRetrievalPreferences:HTTP.Request.ContainerRetrievalPreferences = {
+			include: [
+				NS.C.Class.PreferMembershipTriples
+			],
+			omit: [
+				NS.C.Class.PreferMembershipResources,
+				NS.C.Class.PreferContainmentTriples,
+				NS.C.Class.PreferContainmentResources,
```

Comment:
There's a missing omit preference NS.LDP.Class.PreferMinimalContainer. Please talk with Jorge to make sure that all preferences are listed as needed.